shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1377,6 +1379,12 @@ void updateSemantics(@NonNull ByteBuffer buffer, @NonNull String[] strings) {
                 event.getText().add(object.label);
                 sendAccessibilityEvent(event);
             }
+
+            // If the object is the input-focused node, then tell the reader about it.
+            if (inputFocusedSemanticsNode != null && inputFocusedSemanticsNode.id == object.id) {
+                sendAccessibilityEvent(obtainAccessibilityEvent(object.id, AccessibilityEvent.TYPE_VIEW_FOCUSED));
```

Comment:
Shouldn't we only send this if the focused node changes?

Reply:
Per our conversation: removed this for this PR, and I'll make another that adds it only when the focus changes.